node/test/util/stash_util.js

```diff
@@ -785,9 +786,12 @@ x=E:Cm-1 s=Sa:s;Bm=m;Os Cs foo=bar!Bs=s!W foo=bar`
                 const meta = c.includeMeta;
                 const includeUntracked =
                         undefined === c.includeUntracked || c.includeUntracked;
+                const includeTimestamp =
+                        undefined === c.includeTimeStamp || c.includeTimeStamp;
                 const result = yield StashUtil.makeShadowCommit(
                                                              repo,
                                                              message,
+                                                             includeTimestamp,
```

Comment:
Validate that the timestamp of the resulting commit is the epoch if useEpochTimestamp is used.